include/uadatavariablecache.h

```diff
@@ -62,6 +64,26 @@ class UaPropertyMethodArgument: public UaNode
 
 };
 
+class UaPropertyCache: public UaNode
+{
+public:
+	UaPropertyCache (
+			const UaString  &name,
+			const UaNodeId  &nodeId,
+			const UaVariant &defaultValue,
+			OpcUa_Byte      accessLevel,
+			const UaString& defaultLocaleId);
 
+	virtual UaNodeId nodeId() const { return m_nodeId; }
```

Comment:
_nodeId_ seems to be implicitly an overrider. 
If this is the intention for this method (to be an overrider), it should be flagged with override specifier (or final) rather than virtual.

The same comment stands for the following _typeDefinitionId_, _nodeClass_, _browseName_ though not for _value_.

Reply:
Hi,
did you check if that is well supported by CC7's standard gcc?

Reply:
@parasxos would it satisfy you If we consider this an additional ticket that would kind of focus on the specifier? There are many places where your comment would apply. 
Apart from this, do you have other remarks?

Reply:
No objection on the organization of it. Yeap, I have seen it around is some other places, if you would like to apply it in a catholic way, even better. I don't have any other remarks.
As for CC7 gcc, I don't recall issues when using specifiers. I think though the best is to try.